src/components/editMeasure/populationCriteria/baseConfiguration/BaseConfiguration.test.tsx

```diff
@@ -0,0 +1,291 @@
+import * as React from "react";
+import {
+  render,
+  fireEvent,
+  waitFor,
+  screen,
+  within,
+} from "@testing-library/react";
+import { act } from "react-dom/test-utils";
+import BaseConfiguration from "./BaseConfiguration";
+import useMeasureServiceApi, {
+  MeasureServiceApi,
+} from "../../../../api/useMeasureServiceApi";
+import { Measure } from "@madie/madie-models";
+import userEvent from "@testing-library/user-event";
+import { checkUserCanEdit, measureStore } from "@madie/madie-util";
+
+const mockHistoryPush = jest.fn();
+
+jest.mock("react-router-dom", () => ({
+  ...jest.requireActual("react-router-dom"),
```

Comment:
Since we are already mocking the module, Do we still need to say "requireActual" ?

Reply:
Removed